lib/maths/time_series/unittest/CTimeSeriesModelTest.cc

```diff
@@ -294,7 +294,7 @@ void reinitializeResidualModel(double learnRate,
 
 class CDebug {
 public:
-    static const bool ENABLED{false};
+    static const bool ENABLED{true};
```

Comment:
Did you mean to leave this on for checkin?

If so, please add `results.py` to the `.gitignore` file for this directory so nobody accidentally checks it in.

Reply:
See [d0bdf29](https://github.com/elastic/ml-cpp/pull/2280/commits/d0bdf2951aebdecbd2ebc7a399cf034a5ab2bfba).